src/components/Card/Card.jsx

```diff
@@ -0,0 +1,74 @@
+import React, { Component } from "react";
+import Buttons from "components/Buttons/Buttons";
+import "./style.scss";
+
+class Card extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      id: this.props.item.id,
+      price: this.props.item.price,
+      amount: this.props.item.amount,
+      name: this.props.item.name,
+      img: this.props.item.img,
```

Comment:
why are these 5 fields states? After all, they cannot change, since the constants